### ISSUE TITLE
WavLLM checkpoint

### ISSUE THREAD

**Issue by ming024** [microsoft/SpeechT5#76]
I was trying to download the checkpoints of WavLLM but got the error below
```
<?xml version="1.0" encoding="utf-8"?><Error><Code>AuthenticationFailed</Code><Message>Server failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature.
RequestId:d716386f-801e-0067-504e-8cd1ae000000
Time:2024-04-11T20:24:29.1895155Z</Message><AuthenticationErrorDetail>Signature did not match. String to sign used was r
2024-04-01T02:27:42Z
2025-11-27T02:27:00Z
/blob/valle/share/wavllm/final.pt



2021-10-04
b






</AuthenticationErrorDetail></Error>
```

I guess the Azure sharable link probably expires?

**Comment by XiaoshanHsj** [comment]
The link has been updated

**Comment by BinWang28** [comment]
@ming024  Were you able to inference successfully?

**Comment by ming024** [comment]
@BinWang28 yes sure. I ran inference with a 32G gpu and everything looks great to me now.

**Comment by BinWang28** [comment]
@ming024 Thanks! Let me try again. Never had good luck with fairseq.

**Comment by BinWang28** [comment]
Thanks @ming024. Made it work.

**Comment by berae** [comment]
sir may i ask if i can make it in 1 or2 24GB gpu，oom confusing me for several days

**Comment by berae** [comment]
@BinWang28 hello！may i ask you made it work with a 32G(or more large) gpu?
